kotsadm/web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -1194,6 +1215,27 @@ class AppVersionHistory extends Component {
           </div>
         </Modal>
 
+        {this.state.showDiffErrModal && 
```

Comment:
why not have this condition in `isOpen` instead? that way, the modal does not have to re-render.